tests/base/src/test/resources/ldap/elytron-credential-tests.ldif

```diff
@@ -82,6 +170,17 @@ uid: cryptUser
 userPassword:: e2NyeXB0fWI0Ym5OZVI4VmVjNkk=
 # Password cryptIt
 
+dn: uid=cryptUserCharset,dc=elytron,dc=wildfly,dc=org
```

Comment:
Looks like this one can be removed now.

Reply:
I made a mistake in the comment for this one. It's base64 encoded not hex encoded.

Although these passwords don't support hex encoding, they still support using different charsets. Should I remove this support and the charset tests?

Reply:
Ah, no need to remove this then.